### DISCUSSION TITLE
Server - close a connection

### DISCUSSION THREAD

**Discussion by Dexter0-0** [Links2004/arduinoWebSockets#751] [Q&A]
I tried to find a way for the web sockets server to close a connection with a certain client if a condition is met and did not find how, any ideas on how i could implement this?

Below is a code snippet of exactly what i would like to accomplish

```
void ServerEventHandler(uint8_t ClientID, WStype_t Event, uint8_t * Payload, size_t Length)
{
    switch(Event)
    {
        case WStype_CONNECTED:
            if(LedsServer.remoteIP(ClientID) == <<DESIRED KNOWN LOCAL IP>>)
            {
                HandleNewConnection(Payload, Length);
            }
            else
            {
                // forcfully close the connection with <ClientID>
            }
            break;
    }
}
```

**Comment by noweare1** [comment]
You should be able to use this function  void disconnect(uint8_t num);     num is the client number
I found this in the server header file which list all functions and data types.
